### DISCUSSION TITLE
Can't transition with text-center as base?

### DISCUSSION THREAD

**Discussion by aloh86** [tailwindlabs/tailwindcss#6639] [Help] [answered]
Tailwind v:3.0,5
Using with: Blazor (C#)

`<div class="text-center md:text-right">Some text</div>`

With the above styling, I am finding that it will not transition to text-right on medium breakpoint. However, if I do any of the following:

`<div class="text-left md:text-right">Some text</div>`
`<div class="text-right md:text-left">Some text</div>`
`<div class="text-right md:text-center">Some text</div>`
`<div class="text-left md:text-center">Some text</div>`

...it will work. Tailwind generates correctly:

```
.text-center {
  text-align: center;
}

@media (min-width: 768px) {
    .md\:text-right {
        text-align:f right;
    }
}
```

But for some reason I find that in all browsers (Firefox, Edge, Chrome), the transition from text-center base to text-{left|right} does not work.

For context, the surrounding divs are:

```
<div class="relative">
    <div class="fixed">
        <div>
            <div class="text-center md:text-right">Some text</div>
        </div>
    </div>
    <div class="absolute">
    </div>
</div>
```

**Comment by adamwathan** [comment, marked as answer]
Hey! It does work:

https://play.tailwindcss.com/tWykTUQwYx?size=738x720

If you can create a reproduction in Tailwind Play someone can help you figure out what you need to fix for sure.

**Reply by aloh86** [reply]
Doh! It works. A default bootstrap installation from the project template was interfering with the issue. Sorry for this junk post. Should have investigated a bit more.
